Gemfile

```diff
@@ -50,12 +50,12 @@ gem "bootsnap", require: false
 
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
-  gem "debug", platforms: %i[ mri mingw x64_mingw ]
+  gem 'debug', platforms: %i[mri mingw x64_mingw]
 end
```

Comment:
- [ ]  **Please follow this suggestion and your test will work:**
- Add **gem 'rspec-rails'** in `group :development, :test do`
- run **bundle install** in your terminal
- Then run  **rspec spec/models** to test if the codes has passed
- Then submit for re-review.
- All the best.

Reply:
thank you